HomeTask5.playground/Contents.swift

```diff
@@ -0,0 +1,82 @@
+//1. Дан массив целых чисел. Разделите его на два массива, где в первом массиве будут лежать четные числа, а во втором - нечетные.
+
+import Foundation
+import Darwin
+
+let wholeNumberArray = [1, 2, 3, 4, 5, 6, 10, 14, 16, 19, 21, 34]
+
+let evenNumbersArray = wholeNumberArray.filter({ $0 % 2 == 0 })
+let oddNumbersArray  = wholeNumberArray.filter({ $0 % 2 != 0 })
+
+//2. Дан массив опциональных строк. Посчитайте, сколько букв каждая буква встречается во всех словах.
+
+let optionalStringsArray        = ["A", "Ба", "Ба", "Дельфин", nil]
```

Comment:
You need to put `=` either above other `=` or right after `optionalStringsArray` with one space